angr/state_plugins/unicorn_engine.py

```diff
@@ -1054,6 +1086,19 @@ def setup(self):
         if self.gdt is not None:
             _UC_NATIVE.activate_page(self._uc_state, self.gdt.addr, bytes(0x1000), None)
 
+        if self.state.mode == "tracing":
+            # Pass all concrete fd bytes in native interface to handle relevant syscalls there itself
+            # pylint: disable=attribute-defined-outside-init
+            self.fd_bytes = {}
```

Comment:
Question about this: is there a way to initialize `fd_bytes` dictionary just once? Right now, we will initialize it everytime `setup` is called but that sounds wasteful. I think there are more things that can be initialized just once since they won't ever change(eg: VEX reg ID to unicorn reg ID mapping, blacklisted registers, artificial registers and possibly more).